src/Currency/ArgentinePeso.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace PCF\ValueObject\Currency;
+
+class ArgentinePeso extends AbstractCurrency
+{
+
+    /**
+     * AbstractCurrency constructor.
+     *
+     * @param float $quality
+     */
+    public function __construct($quality)
```

Comment:
u right. but as you noticed, I get all the currences from external source. And code was generated.
I had no input data about subunits. 
In that case, I will have to search in google for it.
for now I think we can close the topic. and come back to currences later